examples/httpx_client.py

```diff
@@ -47,18 +46,18 @@ async def handle_async_request(
         self._http.send_headers(
             stream_id=stream_id,
             headers=[
-                (b":method", method),
-                (b":scheme", url[0]),
-                (b":authority", url[1]),
-                (b":path", url[3]),
+                (b":method", request.method.encode()),
+                (b":scheme", request.url.raw_scheme), 
+                (b":authority", request.url.raw_host),
```

Comment:
Should this be `netloc` or `host`?

Reply:
I'd need to drag through to specs to give you a proper pointer, but... it should be `netloc`.

Same as the `Host` header in HTTP/1.1... https://github.com/encode/httpx/blob/8152c4facd0f71e0f376287e41a0810a60fec9c6/httpx/_client.py#L547